README.md

```diff
@@ -32,10 +32,50 @@ The direction is adjacent to [Adept.ai](https://adept.ai/), with some key differ
 3. OpenAdapt works with all types of desktop GUIs, including virtualized (e.g. Citrix) and web
 4. OpenAdapt is open source! (license TBD, please see https://github.com/MLDSAI/OpenAdapt/issues/246)
 
+
 ## Install
 
+<br/>
+
+| Installation Method  |               [Installation Scripts](https://openadapt.ai/#start)                |                               Manual Setup                                |
+|:--------------------:|:--------------------------------------------------------------------------:|:-------------------------------------------------------------------------:|
+|   Recommended for    |                            Non-technical users                             |                              Technical users                              |
+|     Ease of Use      | Streamlines the installation process for users unfamiliar with setup steps | Allows for more control and customization during the installation process |
+
+<br/>
+
+### Installation Scripts
+
+#### Windows
+- Press Windows Key, type "powershell", and press Enter
+- Copy and paste the following command into the terminal, and press Enter:
+  <pre className="whitespace-pre-wrap code text-slate-600 bg-slate-100 p-3 m-2">
+  Start-Process powershell -Verb RunAs -ArgumentList '-NoExit', '-ExecutionPolicy', 'Bypass', '-Command', "iwr -UseBasicParsing -Uri 'https://raw.githubusercontent.com/MLDSAI/OpenAdapt/main/install/install_openadapt.ps1' | Invoke-Expression"
+  </pre>
+
+#### MacOS
+- Download and install Git and Python 3.10
+- Press Command+Space, type "terminal", and press Enter
+- Copy and paste the following command into the terminal, and press Enter:
+  <pre className="whitespace-pre-wrap code text-slate-600 bg-slate-100 p-3 m-2">
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/OpenAdaptAI/install/HEAD/install_puterbot.sh)"
+  </pre>
```

Comment:
@KrishPatel13 can you please update these instructions to match https://github.com/OpenAdaptAI/OpenAdaptWeb/pull/30/files ?

Reply:
Addressed in: [f8d37ff](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/f8d37ffd3651d164d682203380fa246aea5bc57a)

The website is not uptodate:

<img width="1221" alt="image" src="https://github.com/OpenAdaptAI/OpenAdapt/assets/65433817/f3c7874d-faa0-4c70-88fc-ba6c26c0fcd7">

Reply:
Updated in favour of: https://github.com/OpenAdaptAI/OpenAdaptWeb/pull/30/files#r1252126510 and https://github.com/OpenAdaptAI/OpenAdaptWeb/pull/30/files#r1252128457